### ISSUE TITLE
Full-Screen break window is not focused on any screen when the break timer goes off.

### ISSUE THREAD

**Issue by M0hammedImran** [hovancik/stretchly#689]
### Prerequisites

- [x] I'm using latest version: v 1.1.0
- [x] I've checked [Known issues](https://github.com/hovancik/stretchly#known-issues)
- [x] I've checked [Open issues](https://github.com/hovancik/stretchly/issues)

### Description

[Description of the issue]

Operating system: Windows 10 Build: 19041.450

### Steps to Reproduce

1. Install v:1.1.0
2. Restart the application.
3. and wait for the timer to go off (or skip to the next break).

**Expected behaviour:** Focuses on itself when the timer goes off.

**Actual behaviour:** The break window is behind the current application.

**Reproduces how often:** Every time.

**Comment by hovancik** [comment]
Hi, was this working OK with 1.0? 

How many displays do you have? There's #639

**Comment by M0hammedImran** [comment]
I have two displays and it used to work on both. But now I mostly use just my laptop and it isn't working. It worked great in 1.0.

**Comment by hovancik** [comment]
I haven't changed any code for this, so it must be electron ;/ Multiplatform support seems tobe getting worse and worse. 

I've seen some workaround few people were successful so will try that... Will take some time

**Comment by hovancik** [comment]
Is it for fullscreen apps or regular apps?

**Comment by M0hammedImran** [comment]
The regular windowed view works perfectly. It's just the fullscreen app.

**Comment by anantakrishna** [comment]
> Is it for fullscreen apps or regular apps?

For regular apps. I believe this is a duplicate of #620.

> The regular windowed view works perfectly. It's just the fullscreen app.

I guess you mean that the Stretchly is configured to show breaks in full screen.

**Comment by jsjwooowooo** [comment]
I am doing fullscreen breaks with Stretchly and when I updated I experienced the same issue, my breaks are occurring in the background behind all my open windows.

**Comment by LyubomirGizdov** [comment]
I have the same issue, the break window doesn't appear at all. I can only see a glimpse of it when alt+tabbing. Win10 Pro, build 19041.508. Stretchly breaks are set to full-screen.
It was working with 1.0 (except when I was watching a full screen video, then the break window wouldn't appear).

**Comment by M0hammedImran** [comment]
> 
> 
> I am doing fullscreen breaks with Stretchly and when I updated I experienced the same issue, my breaks are occurring in the background behind all my open windows.

I couldn't have said it better. That is exactly what I was trying to say.

**Comment by hovancik** [comment]
Fullscreen never really worked with Windows, it was always just some workaround, but as it seems they are very unreliable ;/

**Comment by hovancik** [comment]
Hey guys, I can't build on Windows for some reason, but I created branch with simple fix mentioned in Electron https://github.com/electron/electron/issues/20933#issuecomment-685136742. If anyone is running Stretchly from source on Windows, could you try branch `fix/win-fullscreen`?

**Comment by M0hammedImran** [comment]
I am unable to build on Windows 10.

**Comment by BenediktAllendorf** [comment]
TL;DR use `setAlwaysOnTop(true, 'screen-saver')` and everything works! :)

@hovancik yes, that (kind of) fixes the problem (it's even enough to either reduce 1px of width or height), BUT then Windows 10 always shows the taskbar which is a bit ugly.

It played around a bit and apparently the `setAlwaysOnTop()` -method is broken when used within `once('ready-to-show', () => {}`. Not only does it not set the values correctly, it even breaks it. I'm not sure, why it is there in the first place (because that setting is already included in windowOptions), but when removed it works again. Funny enough, even setting it to false leads to the windows being shown (but then you can hide it).

I think I found a general solution, at least for Windows 10: https://github.com/hovancik/stretchly/compare/master...BenediktAllendorf:fullscreen (sorry for the whitespace changes). The [`setAlwaysOnTop()`](https://www.electronjs.org/docs/api/browser-window#winsetalwaysontopflag-level-relativelevel)-method has a level-parameter and setting that to either `normal` or `screen-saver` works (`floating` would be the default and I did not test all other options). Now, it even works when you use another fullscreen up (like videos)! I felt that "screen-saver" fits.

As far as I understand the documentation, it should also be possible to set the `level`/`type` parameter from the start, but that didn't work for me. Also, I haven't found a detailed description of what difference there are exactly (or which side-effects might occur).

**Comment by hovancik** [comment]
@BenediktAllendorf re setAlwaysOnTop: that's electron. I don't remember exact reasons, but probably in past it was working opposite way: if you don't set it via function, it won't work. (many params of BrowserWindow work only via function). Or it only works like it on some OS. Electron is mess in these regards...  

I'll try playing with it.
